components/post_view/post_attachment_container/post_attachment_container.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useHistory} from 'react-router-dom';
+
+export type Props = {
+    className?: string;
+    children?: JSX.Element;
+    link: string;
+};
+const PostAttachmentContainer = (props: Props) => {
+    const {children, className, link} = props;
+    const history = useHistory();
+    const handleOnClick = useCallback((e) => {
+        const tagName = e.target.tagName;
```

Comment:
Since we are going to add an onClick event on posts in https://github.com/mattermost/mattermost-webapp/pull/8342,
maybe we should also `stopPropagation` here.

Reply:
Done!